examples/script_nifti.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
```suggestion
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import os
```

I also like to add the encoding bit to ensure that the correct symbols are used. That can change from different OS systems. By default, it will take it as ASCII, but I always like to specify it so the code doesn't guess it.